lxqt-base/lxqt-session/lxqt-session-0.15.0.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="LXQt session manager"
+HOMEPAGE="https://lxqt.org/"
+
+MY_PV="$(ver_cut 1-2)*"
+
+if [[ ${PV} = *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/lxqt/${PN}.git"
+else
+	SRC_URI="
+		https://downloads.lxqt.org/downloads/${PN}/${PV}/${P}.tar.xz
+		https://github.com/lxqt/${PN}/releases/download/${PV}/${P}.tar.xz
+	"
+	KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86"
+fi
+
+IUSE="+themes +udev"
+
+LICENSE="LGPL-2.1 LGPL-2.1+"
+SLOT="0"
+
+BDEPEND="
+	dev-qt/linguist-tools:5
+	>=dev-util/lxqt-build-tools-0.7.0
+"
+DEPEND="
+	>=dev-libs/libqtxdg-3.3.1
+	dev-qt/qtcore:5
+	dev-qt/qtdbus:5
+	dev-qt/qtgui:5
+	dev-qt/qtwidgets:5
+	dev-qt/qtx11extras:5
+	kde-frameworks/kwindowsystem:5[X]
+	=lxqt-base/liblxqt-${MY_PV}
+	x11-libs/libX11
+	x11-misc/xdg-user-dirs
+	themes? ( =x11-themes/lxqt-themes-${MY_PV} )
```

Comment:
(Is this really needed in DEPEND? And a pure RDEPEND IUSE is banned by QA)
((I wouldn't care though since it's just for themes))

Reply:
It is listed before 'session' in upstream directions here: https://github.com/lxqt/lxqt/wiki/Building-from-source#generalbuild-order

I'll see if I can test it out (I forget if I tested it out in the past already).

Reply:
Yeah, doesn't seem to need to be there at build time at all, or I can't see a reason for it.  I actually don't know why the themes got tied to session in the first place.

Hm.  I guess they could be an unconditional dependency on the meta package or something?  I'm sure there will be someone who doesn't want them being installed...

Reply:
Oh, the lxqt-meta package already does that... heh.